src/main/java/org/folio/circulation/domain/RequestRepresentation.java

```diff
@@ -125,11 +125,11 @@ private static void addAdditionalItemProperties(JsonObject request, Item item) {
     write(itemSummary, "instanceId", item.getInstanceId());
 
 
-    final JsonObject location = item.getLocation();
+    final Location location = item.getLocation();
 
-    if (location != null && location.containsKey("name")) {
+    if (location != null) {
```

Comment:
Please can a unit test be added for when there is no location

Reply:
Do you mean a unit test in the `RequestRepresentationTests` class? If yes the `Item` is always null in this tests. So at first not null `Item` should be covered and then null `Location`.

Reply:
I had meant that, however I hadn't looked too closely into the tests. I'm going to disregard this aspect for the time being.